### DISCUSSION TITLE
About playing when you click the video screen.

### DISCUSSION THREAD

**Discussion by rkselrkswm5** [mifi/lossless-cut#1998] [General]
hello.

In version 3.59.1, the video did not play even when I clicked on the video screen.

In version 3.61.0, click on the video screen to play it.
Is this feature new? Or can I turn it off in the options?

The purpose is to prevent it from playing even if I click on the video screen in the editing screen.

thank you...

**Comment by mifi** [comment]
Hi. it's a new feature. why would you like to turn it off? do you need to click on the video for some other reason?

**Reply by rkselrkswm5** [reply]
hello.

I edit with PotPlayer on the left and losslesscut on the right.
And I am editing it using Corsair's icue macro.

While watching a video in PotPlayer without using the mouse at all, I use icue's keyboard macro to move the mouse cursor across the screen to losslesscut and input the playback time of the video.

However, when moving the screen from PotPlayer to losslesscut, I set it to edit the time after clicking once on the coordinates in the part where the video screen is on the losslesscut screen.
   The coordinates of this part are the same as the coordinates where video time is entered not only in losslesscut but also in other video editing programs, so I clicked on the part where the video was.

As a result, when entering the video time in losslesscut, the video was played, causing inconvenience when entering the correct time.

First of all, since it has been added as a new feature, I will have to find a solution to this.

Thank you.

P.S.
I solved my problem by resetting the screen location of losslesscut and the coordinates where the macro should click, preventing the video from playing.

**Reply by mifi** [reply]
Thanks, I understand. I will see if there are more people who wish the old behaviour back, and if so I might add a setting for this.
